### DISCUSSION TITLE
Branches ahead of MP OpenAPI 3.0 and 3.1

### DISCUSSION THREAD

**Discussion by MikeEdgar** [smallrye/smallrye-open-api#1123] [General]
Proposed branches to support development for MP 3.0 and 3.1:

| Branch | Purpose |
| :------------- | :-- |
| `main` | New features targeting MP 3.1 release (June-ish)  |
| `2.1.x` | Starts at current `main`, on-going maintenance/support. Bug fixes backported from `main` |
| `3.0.x` | Floating branch for MP 3.0 release, periodically reset to head of `2.1.x` and updated with  `to-jakarta.sh` script. Current `jakarta` branch. |

@phillip-kruger , @radcortez - thoughts?

**Comment by phillip-kruger** [comment]
LGTM

**Comment by codespearhead** [comment]
@MikeEdgar This seems to be a solved discussion. [Can you close it](https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions#closing-a-discussion) so that it doesn't show up [in the discussion tab](https://github.com/smallrye/smallrye-open-api/discussions) as unanswered?
